examples/sandbox.py

```diff
@@ -1,21 +1,227 @@
+    income = Account(name="inc", type="INCOME", commodity=curr, parent=b.root_account)
+
+    tr1 = Transaction(post_date=datetime(2015, 10, 21),
+                      description="my revenue",
+                      currency=curr,
+                      splits=[
+                          Split(account=asset, value=(1000, 1)),
+                          Split(account=income, value=(-1000, 1)),
+                      ]
+                      )
+    tr2 = Transaction(post_date=datetime(2015, 10, 25),
+                      description="my expense",
+                      currency=curr,
+                      splits=[
+                          Split(account=asset, value=(-100, 1)),
+                          Split(account=expense, value=(20, 1), memo="cost of X"),
+                          Split(account=expense, value=(80, 1), memo="cost of Y"),
+                      ]
+                      )
+    tr_stock = Transaction(post_date=datetime(2015, 10, 29),
+                           description="my purchase of stock",
+                           currency=curr,
+                           splits=[
+                               Split(account=asset, value=(-200, 1)),
+                               Split(account=expense, value=(15, 1), memo="transaction costs"),
+                               Split(account=stock, value=(185, 1), quantity=(6, 1), memo="purchase of stock"),
+                           ]
+                           )
+    tr_to_foreign = Transaction(post_date=datetime(2015, 10, 30),
+                                description="transfer to foreign asset",
+                                currency=curr,
+                                splits=[
+                                    Split(account=asset, value=(-200, 1)),
+                                    Split(account=foreign_asset, value=(200, 1), quantity=(135, 1)),
+                                ]
+                                )
+    tr_from_foreign = Transaction(post_date=datetime(2015, 10, 31),
+                                  description="transfer from foreign asset",
+                                  currency=other_curr,
+                                  splits=[
+                                      Split(account=asset, value=(135, 1), quantity=(215, 1)),
+                                      Split(account=foreign_asset, value=(-135, 1)),
+                                  ]
+                                  )
+    Price(commodity=cdty,
+          currency=other_curr,
+          date=datetime(2015, 11, 1),
+          value=(123, 100),
+          )
+    Price(commodity=cdty,
+          currency=other_curr,
+          date=datetime(2015, 11, 4),
+          value=(127, 100),
+          )
+    Price(commodity=cdty,
+          currency=curr,
+          date=datetime(2015, 11, 2),
+          value=(234, 100),
+          )
+
+    b.save()
+
+    print(b.prices_df().to_string())
+    print(b.prices)
+
+fdsdfsfds
```

Comment:
Hey @sdementen , what are these chars? Are they splitting this example somehow? There are more occurrences in this file